docs/documentation.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started
```

Comment:
I'm a bit torn about this one.  How about an alternative where we bust up the README so that it's really short and contains a link to these docs?  Each section could be smaller and easier to consume.  e.g. there could be an entire page devoted to how to do dev on this project.

Alternatively, mkdocs could just be a rendering mechanism for API documentation and symlink a huge readme.

My preference is one or the other.